tracks/ruby/exercises/resistor-colors/mentoring.md

```diff
@@ -0,0 +1,33 @@
+Resistor Colors is a side exercise, unlocked by Hello World
+
+### New Concepts
+Array, Array#index, `map`, Constant, chaining methods
+
+### Minimal solution for approval
+```ruby
+
+class ResistorColors
+
+  COLOR_CODES = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
+  
+  def self.value(colors)
+    colors.map { |color| COLOR_CODES.index(color) }.join.to_i
+  end
+
+end
+
+```
+
+### General
+This is the very first Array exercise, and the first loop. Introducing `map` is key in this exercise.
```

Comment:
> This is the very first Array exercise

Isn't this set up as a "bonus" exercise? ([reference](https://github.com/exercism/ruby/pull/939/commits/3ccf5c6211c57da699ea18c15d51537f5eb65663))